src/redux/books/books.js

```diff
@@ -1,17 +1,25 @@
 const ADD_BOOK = 'bookstore/books/ADD_BOOK';
 const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+const LOAD_BOOKS = 'bookstore/books/LOAD_BOOKS';
+const APP_ID = 'W2TAgTVlPqLEGcqZHoYR';
 
 const defaultState = [];
 
 export default function books(state = defaultState, action) {
   switch (action.type) {
+    case LOAD_BOOKS:
+      return action.payload.map((bookObject) => ({
+        ...bookObject,
+        author: 'Suzanne Collins',
+        progress: 74,
```

Comment:
- [ ] The keys & values for the **bookObject** ca be placed in the **defaultState** instead.